src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj

```diff
@@ -127,6 +127,27 @@
     </EmbeddedResource>
     <EmbeddedResource Include="ProtoGeometryImages.resx" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
seems incorrect that these files are embedded directly in the proj file - aren't the rest of them just embedded into the resx? could be wrong here.

Reply:
@Racel Did you embed the resources you added as well?  You should only have to embed the icon after it is added to the resx not the original resources.

Reply:
To clarify:
- add the resources
- drag them into the appropriate .resx
- select the icon you just drop in the .resx (not the original resource)
- embed in the properties window

Reply:
@alfarok - i guess i was embedding the resource, but not sure how to select just the icon...